src/test/resources/com/datadog/api/v1/client/api/aws_logs_integration.feature

```diff
@@ -5,34 +5,40 @@ Feature: AWS Logs Integration
   page](https://docs.datadoghq.com/api/?lang=bash#integration-aws-logs).
 
   Background:
-    Given a valid "apiKeyAuth" key in the system
-    And a valid "appKeyAuth" key in the system
-    And an instance of "AWSLogsIntegration" API
+    Given an instance of "AWSLogsIntegration" API
 
   @generated @skip
   Scenario: Add AWS Log Lambda ARN returns "Bad Request" response
-    Given new "CreateAWSLambdaARN" request
+    Given a valid "apiKeyAuth" key in the system
+    And a valid "appKeyAuth" key in the system
+    And new "CreateAWSLambdaARN" request
```

Comment:
What happened here ?

Reply:
It looks like `CheckAWSLogsLambdaAsync` is missing `apiKeyAuth/appKeyAuth` in OpenAPI spec.